tutor/specs/e2e/assignment-review.e2e.ts

```diff
@@ -51,30 +51,204 @@ test.describe('Assignment Review', () => {
         await page.click('testId=confirm-delete-assignment')
         await page.waitForSelector('tourRegion=teacher-calendar')
     })
 
+    test.describe('in a different timezone', () => {
+        test.use({ timezoneId: 'America/New_York' });
+
+        test('after creating assignment it edits and can delete', async ({ page }) => {
+            const assignmentName = faker.hacker.phrase()
+            await visitPage(page, `/course/${COURSE_ID}/assignment/edit/reading/new`)
+            await page.fill('testId=edit-assignment-name', assignmentName)
+            await page.click('text="Save & Continue"')
+            await page.click('text="Problem-Solving Strategies"')
+            await page.click('text="Save & Continue"')
+            await page.click('text="Save as Draft"')
+
+            await page.waitForSelector(`tourRegion=teacher-calendar >> text="${assignmentName}"`)
+
+            await page.click(`text="${assignmentName}"`, { position: { x: 33, y: 8 } })
+            await page.waitForSelector('.tasking-date-time')
+
+            await page.waitForSelector('tourRegion=reading-assignment-editor')
+            expect(
+                await page.evaluate(() => document.location.pathname)
+            ).toMatch(/assignment\/edit\/reading/)
+
+            await page.click('.opens-at')
+            await page.click('.oxdt-dropdown:not(.oxdt-dropdown-hidden) .oxdt-cell-today')
+            await page.click('.oxdt-dropdown:not(.oxdt-dropdown-hidden) button')
+
+            await page.click('text="Save & Continue"')
+
+            await page.click('text="Save & Continue"')
+
+            await page.click('text="Publish"')
+
+            await page.waitForSelector(
+                `tourRegion=teacher-calendar >> css=.is-published.is-open >> text="${assignmentName}"`
+            )
+
+            await page.click(`text="${assignmentName}"`, { position: { x: 33, y: 8 } })
+
+            const detailsFormat = 'ccc, MMM d, h:mm a z' // TimeHelper.HUMAN_DATE_TIME_TZ_FORMAT
+            const oldDueAt = DateTime.fromFormat(
+                await page.$eval(
+                    '.tasking-date-time.row + .tasking-date-time.row .due-at',
+                    node => (node as HTMLElement).innerText
+                ), detailsFormat
+            )
+            const oldClosesAt = DateTime.fromFormat(
+                await page.$eval(
+                    '.tasking-date-time.row + .tasking-date-time.row .closes-at',
+                    node => (node as HTMLElement).innerText
+                ), detailsFormat
+            )
+
+            await page.click('text="View assignment"')
+            await page.click('testId=edit-assignment')
+
+            await page.click('.due-at')
+            await page.click(
+                '.oxdt-dropdown:not(.oxdt-dropdown-hidden) .oxdt-cell-selected + .oxdt-cell'
+            )
+            await page.click('.oxdt-dropdown:not(.oxdt-dropdown-hidden) button')
```

Comment:
some of these selectors you're using a lot, you could put the selectors in variables for readability and maintainability